client/me/help/help-contact/index.jsx

```diff
@@ -146,6 +147,7 @@ class HelpContact extends React.Component {
 
 		this.props.sendUserInfo( howCanWeHelp, howYouFeel, site );
 		this.props.sendHappychatMessage( message, { includeInSummary: true } );
+		this.props.sendNotTyping( message );
```

Comment:
Do we need to send `not typing` whenever a message is sent? Could we maybe improve on that?

Reply:
Good catch! I was so focused on not modifying the typing behavior in place, that I didn't realize this wasn't necessary. Fixed.